docs/source/tutorials.ipynb

```diff
@@ -38,9 +38,16 @@
      "maxdepth": 1
     }
    },
-    "source": [
+   "source": [
     "[Using the Cost-Benefit Matrix Objective](demos/cost_benefit_matrix)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
```

Comment:
@eccabay I think you have to update this to be
```python
   "metadata": {
    "nbsphinx-toctree": {
     "maxdepth": 1
    }
   },
```
in order to add this entry to the sphinx toctree